### PR TITLE
Add batman to the adventure

### PR DESCRIPTION
**Pull request by aleliszt**

### CODE REVIEW

**Comment by aleliszt** [comment]
Please review this, @casey-collab

File: espanol/batman/Batman.md [hunk 1]
```diff
@@ -0,0 +1 @@
+No puedes ser Batman cuando yo soy el verdadero Batman, sigue y se un [Robin] (robin/robin.md)
```
Comment (casey-collab) [inline espanol/batman/Batman.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: espanol/robin/robin.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Nadie quiere ser Robin, mejor vuelve al inicio de [todo] (canudos.md)
```
Comment (casey-collab) [inline espanol/robin/robin.md]: This link is also invalid.

File: espanol/batman/Batman.md (hunk 1 above)
Comment (casey-collab) [inline espanol/batman/Batman.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by aleliszt** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**